web/src/lib/library/diff.ts

```
@@ -80,6 +101,53 @@ export const deriveMutationFromDifference = (
+      case "url":
+        if (!isValidLinkLike(updatedValue as string)) {
+          console.debug(
+            "Skipping mutation for 'url' because it is not valid",
+            updatedValue,
+          );
+          return;
+        }
+        break;
+
+      case "children":
+        const updatedChildren = updatedValue as NodeWithChildren["children"];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)